shell/packages/sandstorm-db/db.js

```diff
@@ -843,6 +894,30 @@ _.extend(SandstormDb.prototype, {
     return user && Plans.findOne(user.plan || "free");
   },
 
+  getMyReferralBonus: function(user) {
```

Comment:
FWIW I'm doing my best to copy the logic of `getMyUsage`! Maybe we should discuss this bit IRL, but I can also take detailed suggestions here.